skimage/restoration/deconvolution.py

```diff
@@ -364,7 +364,9 @@ def richardson_lucy(image, psf, num_iter=50, clip=True, filter_epsilon=None):
     Parameters
     ----------
     image : ndarray
-       Input degraded image (can be n-dimensional).
+       Input degraded image (can be n-dimensional). Input image
+       should be normalized to the range [0, 1] to allow for proper
+       floating point conversion prior to deconvolution.
```

Comment:
```suggestion
       Degraded image (can be n-dimensional). This input image
       should be normalized to the range [0, 1] or [-1,1] if you keep
       the default `clip=True`. If you want to use `clip=False`, you may
       pass a non-normalized image.
```
@lagru @stefanv should we add a sentence to announce that this behaviour will be changed / fixed in skimage2 / the future?

Reply:
Hi @mkcor, thanks for helping. I just pushed some minor changes based on your suggestions.
Let me know what else needs doing, I'm all ears!